packages/api/emails/survey/html.pug

```diff
@@ -38,8 +38,25 @@ html
                 border: none;
                 padding: 1.5rem 2rem;
             }
+
+            .button-styles {
+                background: #128b3e;
+                color: white;
+                font-weight: bold;
+                border-style: smooth;
+                border-radius: 1rem;
+                border-color: #128b3e;
+                padding: 1rem;
+
+            }
+
+            .button-styles:hover {
+              background: #6d7278;
+              transition: background 0.2s ease-in-out;
```

Comment:
The indentation here is a little wonky, I think a couple more spaces over would bring it into line with the rest of the file.